bindings/c/test/mako/mako.c

```diff
@@ -1657,7 +1682,11 @@ int parse_args(int argc, char* argv[], mako_args_t* args) {
 			args->api_version = atoi(optarg);
 			break;
 		case 'c':
-			strcpy(args->cluster_file, optarg);
+			strcpy(args->cluster_files[args->num_fdb_clusters], optarg);
```

Comment:
How do you guarantee `num_fdb_clusters` is already set properly when it reaches here?